tools/conventional-changelog-spectrum/LICENSE.md

```diff
@@ -0,0 +1,15 @@
+### ISC License
```

Comment:
Not sure what to do about the fact that this was forked from https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular. We need to reach out to that project's owners and see if this needs to live in their monorepo...